src/utils.ts

```diff
@@ -22,11 +24,16 @@ export async function executeCallOrSend(
     const selectedRpcUrl = rpcOracle.getNextAvailableRpc();
 
     try {
+      const start = performance.now();
       const result = await rpcProviderFn(
         isOptimismOrBaseNetwork(String(networkId))
           ? asL2Provider(new ethers.providers.JsonRpcProvider(selectedRpcUrl))
           : new ethers.providers.StaticJsonRpcProvider(selectedRpcUrl),
       );
+      const end = performance.now();
+      const kafkaManager = KafkaManager.getInstance();
+      if (kafkaManager) kafkaManager.sendRpcResponseTimeToKafka(selectedRpcUrl, end - start, requestId);
```

Comment:
ethers.js does not provide a tracking response time feature, 
so use NodeJs perf_hooks 
https://medium.com/@priteymehta9/perf-hooks-in-nodejs-303fd1c8bfb7